cypress/e2e/sample_test_spec/SampleTestRequest.cy.ts

```diff
@@ -0,0 +1,42 @@
+import { SampleTestPage } from "pageobject/Sample/SampleTestCreate";
+
+describe("Sample Test", () => {
+  const sampleTestPage = new SampleTestPage();
+  const sampleTestType = "BA/ETA",
+    icmrCategory = "Cat 0",
+    icmrLabel = "Test Icmr Label";
+
+  before(() => {
+    cy.loginByApi("devdistrictadmin", "Coronasafe@123");
+    cy.saveLocalStorage();
+  });
+
+  beforeEach(() => {
+    cy.restoreLocalStorage();
+    cy.clearLocalStorage(/filters--.+/);
+  });
+
+  it("should request a new sample test", () => {
+    sampleTestPage.visitPatientPage();
+    sampleTestPage.visitPatientDashboardPage();
+    sampleTestPage.visitSampleRequestPage();
+
+    // Fill form fields
+    sampleTestPage.selectSampleType(sampleTestType);
+    sampleTestPage.selectIcmrCategory(icmrCategory);
+    sampleTestPage.typeIcmrLabel(icmrLabel);
+
+    // Submit the form
+    sampleTestPage.submitForm();
+
+    // Check for sample request notification and history
+    sampleTestPage.clickOnNotification();
+    sampleTestPage.checkRequestHistory();
```

Comment:
```suggestion
    sampleTestPage.checkRequestHistory();
```

- this should verify the content on the card present in the patient details page, where status , sample type and result is displayed.
- we have reusable component for verifying content presence, refer to commands.ts file